### PR TITLE
Add plane lidar model for testing fw terrain landing

### PR DESCRIPTION
**Pull request by Jaeyoung-Lim**
**Describe problem solved by this pull request**
This PR adds a distance sensor to the plane model to test terrain following with fixed wing

### CODE REVIEW

**Comment by julianoes** [comment]
@Jaeyoung-Lim wow you're fast :smile:!

Should this be merged or are you planning to add more things?

**Comment by Jaeyoung-Lim** [comment]
@julianoes haha this PR is missing the necessary parameter changes for terrain following. I have synced with @dagar and will add them to this PR

File: ROMFS/px4fmu_common/init.d-posix/1033_plane_lidar [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+#
+# @name Plane SITL with catapult
```
Comment (TSC21) [inline ROMFS/px4fmu_common/init.d-posix/1033_plane_lidar]:
```suggestion
# @name Plane SITL with downward facing LIDAR.
```

File: ROMFS/px4fmu_common/init.d-posix/1033_plane_lidar [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+#
+
+sh /etc/init.d-posix/1030_plane
+
+if [ $AUTOCNF = yes ]
+then
+	param set RWTO_TKOFF 0
+
```
Comment (TSC21) [inline ROMFS/px4fmu_common/init.d-posix/1033_plane_lidar]: So where are the specific configurations to use the LIDAR and the distance to bottom?

Comment (TSC21) [reply ROMFS/px4fmu_common/init.d-posix/1033_plane_lidar]: Ah I see you will be working on that (https://github.com/PX4/Firmware/pull/14480#issuecomment-604274564).

**Comment by hamishwillee** [comment]
What docs will this need? 
Specifically, currently the docs say we only support terrain following in MC mode: https://docs.px4.io/master/en/flying/terrain_following_holding.html

File: ROMFS/px4fmu_common/init.d-posix/1033_plane_lidar (hunk 2 above)
Comment (Jaeyoung-Lim) [reply ROMFS/px4fmu_common/init.d-posix/1033_plane_lidar]: @TSC21 Done!

**Comment by Jaeyoung-Lim** [comment]
@hamishwillee Actually, I realized that this part of the firmware is completely undocumented. But it exists in the code. I think it is a good opportunity that we document this. 

Also, I think this is also different with the terrain following - this only enables using terrain information on landing. @dagar Please correct me if I am wrong

**Comment by hamishwillee** [comment]
> @hamishwillee Actually, I realized that this part of the firmware is completely undocumented. But it exists in the code. I think it is a good opportunity that we document this.

Absolutely! Are you happy to do the first pass? 

Hard to know exactly the right place for this. Probably in https://docs.px4.io/master/en/flying/fixed_wing_landing.html (which mentions only terrain from GPS). Then maybe a small section and cross link from https://docs.px4.io/master/en/flying/terrain_following_holding.html ?

File: ROMFS/px4fmu_common/init.d-posix/1033_plane_lidar (hunk 2 above)
Comment (TSC21) [reply ROMFS/px4fmu_common/init.d-posix/1033_plane_lidar]: Where exactly? I still see the same params.

**Comment by Jaeyoung-Lim** [comment]
Since https://github.com/PX4/sitl_gazebo/pull/447 was merged, updated submodule 

@hamishwillee Sure!

File: ROMFS/px4fmu_common/init.d-posix/1033_plane_lidar (hunk 2 above)
Comment (Jaeyoung-Lim) [reply ROMFS/px4fmu_common/init.d-posix/1033_plane_lidar]: I added `param set FW_LND_USETER 1` do you see otherwise?

**Comment by dagar** [comment]
Fix the merge conflict (submodule changed) and merge?

**Comment by Jaeyoung-Lim** [comment]
@dagar Done!

**Comment by Jaeyoung-Lim** [comment]
@dagar Can we get this merged please?

**Pull request merged**